lunes_cms/cms/admins/discipline_admin.py

```diff
@@ -274,6 +274,15 @@ def make_unreleased(self, request, queryset):
         ordering="modules_released_order",
     )
     def modules_released(self, obj):
+        """
+        returns HTML Tag of the link to released training sets related to the discipline
+
+        :param obj: Discipline object
+        :type obj: models.Discipline
```

Comment:
```suggestion
        :type obj: ~lunes_cms.cms.models.discipline.Discipline
```
Sphinx always wants the whole module path in order to perform the linking.